xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-api/src/main/java/org/xwiki/model/internal/reference/DefaultSymbolScheme.java

```diff
@@ -178,14 +188,29 @@ public class DefaultSymbolScheme implements SymbolScheme
      */
     public DefaultSymbolScheme()
     {
-        initialize();
+        initialize(false);
     }
 
     /**
      * Initialize internal data structures.
+     * 
+     * @param withParameters true if the parameters syntax should be supported for all entity types
      */
-    public void initialize()
+    public DefaultSymbolScheme(boolean withParameters)
     {
+        initialize(withParameters);
+    }
+
+    /**
+     * Initialize internal data structures.
+     * 
+     * @param withParameters true if the parameters syntax should be supported for all entity types
+     */
+    public void initialize(boolean withParameters)
```

Comment:
This one does not need to be public since it's controlled by the constructor no?

Reply:
Then I'd make it private since the previous initialize was removed anyway.